src/components/WebexRemoteMedia/WebexRemoteMedia.js

```diff
@@ -16,22 +16,30 @@ import './WebexRemoteMedia.scss';
  * NOTE: waiting for the UX for a design on what to display if there is no remote video
  */
 export default function WebexRemoteMedia({meetingID}) {
-  const {remoteAudio, remoteVideo} = useMeeting(meetingID);
+  const {remoteAudio, remoteVideo, error} = useMeeting(meetingID);
   const audioRef = useStream(remoteAudio);
   const videoRef = useStream(remoteVideo);
   const hasMedia = !!(remoteAudio || remoteVideo);
 
   return (
-    <div className={`${WEBEX_COMPONENTS_CLASS_PREFIX}-remote-media`}>
-      {!hasMedia ? (
-        <Badge rounded>
-          <Spinner size={16} />
-          <div>Connecting</div>
-        </Badge>
-      ) : null}
-      {remoteVideo ? <video ref={videoRef} playsInline autoPlay /> : null}
-      {remoteAudio ? <audio ref={audioRef} autoPlay /> : null}
-    </div>
+    <React.Fragment>
+      {error ? (
+        <AlertBanner show type="warning">
```

Comment:
I noticed the banner is outside the `wxc-remote-media` div. Everything in the component should be encapsulated within the main component div. Not sure if you tried it outside of Storybook, but without the container box styling of `wxc-remote-media` the remote media component would only take the space of the banner wherever it's placed, instead of taking the full size of what the parent element allows it

Reply:
I've fixed it with the approach that we discussed. Can you review @lalli-flores ? Thanks